sdks/universal-router-sdk/test/mixedTrades.test.ts

```diff
@@ -217,5 +217,5 @@ describe('SwapRouter.swapCallParameters', () => {
       registerFixture('_ERC20_FOR_NFTS_PARTIAL_FILL', methodParameters)
       expect(methodParameters.value).to.eq('0x00')
     })
-  })
+  }).timeout(100000)
```

Comment:
Is this and the hardhat.config changes needed?
mocha: {
    timeout: 100000,
  },

Reply:
yeah after adding it in hardhat.config, this inline change is probably not needed anymore